src/tooltip.js

```diff
@@ -133,21 +137,44 @@ angular.module('ts.utils')
           }
         }
 
-        function makeVisible(){
-          if(!isVisible) {
-            document.body.insertBefore(tooltipContainer[0],document.body.childNodes[0]);
+        function makeVisible() {
+          if (!isVisible) {
+            document.body.insertBefore(tooltipContainer[0], document.body.childNodes[0]);
             positionTooltip();
             isVisible = true;
+            if($scope.tsTooltipContentHover) {
+              angular.element(tooltipContainer).bind('mouseleave', function(){
```

Comment:
yea they do. When you remove something from the dom, the listeners are destroyed as well